demo/mobx.jsx

```diff
@@ -1,7 +1,6 @@
 import React, { forwardRef, useRef, useState } from 'react';
 import { decorate, observable } from 'mobx';
 import { observer, useObserver } from 'mobx-react';
-import 'mobx-react-lite/batchingForReactDom';
```

Comment:
It's been a console warning only for 5+ years, no reason to keep it:

https://github.com/mobxjs/mobx/blob/main/packages/mobx-react-lite/batchingForReactDom.js